Cython/Compiler/Code.py

```diff
@@ -2374,6 +2414,39 @@ def intern_identifier(self, text):
     def get_cached_constants_writer(self, target=None):
         return self.globalstate.get_cached_constants_writer(target)
 
+    def name_in_module_state(self, cname):
+        # If we don't have a funcstate scope assume global scope for now
+
+        # default fallback option if we don't know better
+        modulestate_name = Naming.modulestateglobal_cname
+
+        # self.funcstate can be None in a few places, including ToPyStructUtilityCode.
+        # TODO - this will eventually need to be fixed.
```

Comment:
Agreed. There's usually not a good reason for difference like this. They always get in the way at some point.

Reply:
Should hopefully be gone (at least in ToPyStructUtilityCode)